pip.md

```diff
@@ -40,20 +40,25 @@ RAPIDS users can once again install RAPIDS via pip!  This is an **experimental r
 
 > <i class="fas fa-desktop text-white"></i> **OS:** One of the following OS versions:
 
->> <i class="fa-brands fa-ubuntu text-white"></i> Ubuntu 18.04/20.04 or CentOS 7 / Rocky Linux 8 with <code>gcc/++</code> 9.0+
-
->> <i class="fas fa-desktop text-white"></i> Windows 11 using WSL2  **[See separate install guide <i class="fa fa-angle-double-right" aria-hidden="true"></i>](wsl2.html){: target="_blank"}**
-
+>> <i class="fa-brands fa-ubuntu text-white"></i> Ubuntu 18.04/20.04 or CentOS 7 / Rocky Linux 8 with <code>gcc/++</code> 9.0+  <br>
+>> <i class="fas fa-desktop text-white"></i> Windows 11 using WSL2  **[See separate install guide <i class="fa fa-angle-double-right" aria-hidden="true"></i>](wsl2.html){: target="_blank"}** <br>
 >>> <i class="fas fa-chevron-circle-right text-white"></i> In addition, WSL2 pip installations require following **[this guide to install the CUDA Toolkit without drivers.](https://docs.nvidia.com/cuda/wsl-user-guide/index.html#cuda-support-for-wsl2){: target="_blank"}**
 
-> <i class="fas fa-info-circle text-white"></i> **Glibc version:** x86_64 wheels require glibc >= 2.17 and ARM architecture (aarch64) wheels require glibc >= 2.31 (only ARM Server Base System Architecture is supported).
+> <i class="fas fa-microchip text-white"></i> **Architecture:** One of the following Architectures:<br>
+>> <i class="fas fa-chevron-circle-right text-white"></i> x86_64 <br>
+>> <i class="fas fa-chevron-circle-right text-white"></i> **[ARM SBSA](https://developer.arm.com/documentation/102378/0201/Armv8-x-A-and-the-SBSA){: target="_blank"}**.  Jetson support coming soon!
```

Comment:
I would suggest we don't commit to Jetson support coming soon unless we've solidified a clear plan for official support.

Reply:
Had left a similar (now deleted) comment. There's a lot of unknowns here and it's definitely not happening until 23.02 at the earliest so if nothing else I'd pull the 'Soon'. 

Are all Jetson's on the same arch? Would we only support some? Just a couple questions that came to mind too. Jetson might deserve its own page in the future, or at least announcement.

Reply:
I can say "Jetsons not supported yet".  I've got my Orin, so it will happen hopefully before Christmas